tooling/FST-1005-package-signing.md

```diff
@@ -1,16 +1,22 @@
-# FS-1005 Package Signing
+# FS-1005 Package Authoring for Microsoft co-owned packages
 
-This is inspired by [this PR](https://github.com/Microsoft/visualfsharp/pull/4734).
+As per the requirements by Microsoft's Developer Division, all NuGet packages which are developed and distributed by Microsoft (including if they are co-owned) must be adhere to the following:
 
-As per the requirements by Microsoft's Developer Division, all NuGet packages which are developed and distributed by Microsoft (including if they are co-owned) must be signed with Microsoft's signing keys. This includes the following packages:
+* Co-owned on [nuget.org](https://www.nuget.org/) by the [Microsoft](https://www.nuget.org/profiles/microsoft) and [fsharporg](https://www.nuget.org/profiles/fsharporg) organizations.
+* Binary is signed with Microsoft signing keys.
+* Package itself is signed with Microsoft signing keys.
+* The following package authoring:
+    * **Author** includes "Microsoft".
+    * **Owner** includes "Microsoft".
```

Comment:
License URL and Requiring License Acceptance
The License URL must be a valid product license URL and the ​"RequireLicenseAcceptance" property must be set to true.

​Project URL
The Project Url must be a valid team or product Url, and it can be a FWLink.  This must be a permalink that will never be deprecated.